src/Blocks/Input.purs

```diff
@@ -0,0 +1,28 @@
+module CN.UI.Block.Input (input) where
+
+import Prelude
+
+import DOM.HTML.Indexed (HTMLinput)
+import Halogen.HTML as HH
+import Halogen.HTML.Properties as HP
+
+inputClasses :: Array HH.ClassName
+inputClasses = HH.ClassName <$>
```

Comment:
How does this work if we have different kinds of inputs? Say a 'disabled' input, an input with a different background somewhere in the app, etc. -- what does customization look like?

Reply:
In each case, we'd need to have a data type with variants possible for the field as with Button?

Reply:
Not sure yet - I am being judicious about how generic these blocks are. I am hoping that building some the forms will highlight gaps.